src/tailscale/cli.ts

```diff
@@ -100,42 +100,109 @@ export class Tailscale {
         throw new Error('childProcess.stdout is null');
       }
 
-      if (this.childProcess.stderr) {
-        let buffer = '';
-        this.childProcess.stderr.on('data', (data: Buffer) => {
-          buffer += data.toString(); // Append the data to the buffer
+      this.processStderr(this.childProcess);
+    });
+  }
 
-          const lines = buffer.split('\n'); // Split the buffer into lines
+  processStderr(childProcess: cp.ChildProcess) {
+    if (childProcess.stderr) {
```

Comment:
nit: We can return early with the else clause here and avoid the wrapped `if/else` block.

Reply:
This code already existed, just got moved to its own method. It doesn't return, it throws 😃 but yeah will just have that be done at the top 👍🏼